app/templates/auth/request-password-reset.html

```diff
@@ -3,23 +3,19 @@
 {% block page_title %}Reset password – Digital Marketplace{% endblock %}
 
 {% block breadcrumb %}
-  {%
-    with items = [
-      {
-        "link": "/",
-        "label": "Digital Marketplace"
-      },
-      {
-        "link": "/user/login",
-        "label": "Login"
-      },
-      {
-        "label": "Reset password"
-      }
-    ]
-  %}
-    {% include "toolkit/breadcrumb.html" %}
-  {% endwith %}
+  {{ govukBreadcrumbs({"items": [
+    {
+      "text": "Digital Marketplace",
+      "href": "/",
+    },
+    {
+      "text": "Login",
+      "href": "/user/login",
+    },
+    {
+      "text": "Reset password",
```

Comment:
Could this be "Request password reset" or "Request new password"?

Looking at the breadcrumbs I thought it was part of the change password user journey. It was only after looking at the filename did I realise that this was meant for user who had forgotten their password.

Reply:
@pocketstefan!!

Reply:
After discussion with @pocketstefan, we agreed to keep this the same 👍